lang/py3/avro/tests/test_protocol.py

```diff
@@ -453,14 +453,14 @@ def testValidCastToStringAfterParse(self):
       proto = protocol.Parse(example.protocol_string)
```

Comment:
This whole test is pretty weird -- we first parse the example.protocol_string, and then we parse the result of that _again_? @Fokko do you know if this test is right at all (even before my changes)?